### ISSUE TITLE
Base font change from within the .style file

### ISSUE THREAD

**Issue by GiannisGG** [MacDownApp/macdown#994]
Hello,

This is not really an "Issue," but a question:

I created a new .style file, with a theme that I fancy, by duplicating the Tommorow+ theme and changing font sizes. My question is: **how can I change the base font from within the .style file,** rather than from *Preferences>Editor* ?

Is there a glossary for the language used in .style files?

Thank you,
G.

**Comment by GiannisGG** [comment]
Anybody?

**Comment by brandondurham** [comment]
I don’t believe it’s possible. Not that I can find in any other *.style file.
